.github/nfpm-configs/nfpm-linux-ziti-cli.yaml

```diff
@@ -0,0 +1,20 @@
+# nfpm configuration file
+#
+# check https://nfpm.goreleaser.com/configuration for detailed usage
+#
+name: ziti-cli
+arch: ${GOARCH}
+platform: linux
+version: ${ZITI_VERSION}
+maintainer: ${ZITI_MAINTAINER}
+description: >
+  The ziti-cli package provides the ziti executable binary as a command line
+  interface for Ziti.
+vendor: ${ZITI_VENDOR}
+homepage: ${ZITI_HOMEPAGE}
+license: Apache-2.0
+contents:
+- src: ./release/ziti
+  dst: /usr/bin/ziti
```

Comment:
Should this install to `/opt/openziti/bin` instead?

Reply:
Only if we symlink it from a dir that's typically in the executable search PATH because the CLI is user facing and interactive, not a daemon. That's more a UX perspective than a rigid adherence to FHS, but I can't imagine it'd go over well asking every user to modify their PATH.